docs/developer-guide/getting-started.md

```diff
@@ -47,7 +47,7 @@ make lua-test
 Lua tests are located in the directory `rootfs/etc/nginx/lua/test`
 
 !!! important
-    Test files must follow the naming convention `<mytest>_test.lua` or it will be ignored
+    Test file must follow the naming convention `<mytest>_test.lua` or it will be ignored
```

Comment:
why this change? here we are saying that all the files, so the original is right

Reply:
I think `test file` may be better consistent with `it` later.Yes,any one of them should meet this condition,or the test file will be ingored.